terraform/production/main.tf

```diff
@@ -87,6 +87,18 @@ module "application" {
   sidekiq_task_desired_count = 2
   sidekiq_task_memory        = 8192
 
+  out_of_hours_scale_down = {
+    enabled = true
+    timings = {
+      workday_start = "0 7"
+      workday_end   = "30 19"
+    }
+    scale_to = {
+      app     = 2
+      sidekiq = 2

```

Comment:
Could there be a situation where something long is running on sidekiq? I guess it would just fail and restart on another instance, but is there a way to avoid shutting down busy instances (assuming this is not doing that already)?

Reply:
Sidekiq is already set to 2 for prod, so this doesn't actually change it (otherwise you're right I'd be a bit worried we'd lose ongoing work for a long running task even though it would be restarted on another instance - should probably find a way to make that less of a problem but I suspect that may be a bit tricky to do)